pkg/cmd/pipelinerun/list.go

```diff
@@ -156,7 +159,11 @@ func list(p cli.Params, pipeline string, limit int, labelselector string) (*v1al
 		}
 	}
 
-	prc := cs.Tekton.TektonV1alpha1().PipelineRuns(p.Namespace())
+	ns := p.Namespace()
```

Comment:
We should probably have some validation when a user specified `-n` but also `--all-namespaces`. I think adding an error if a user does this saying the flags are incompatible would be helpful. WDYT?

Reply:
No I think this should be fine. While writing the tests I noticed that in pipeline run if we don't pass `-n` with the namespace the command has to run in context with, the test fails with error regarding namespaces provided in the test.

Reply:
Can you elaborate on what test fails?